pkgs/applications/science/misc/openmvg/default.nix

```diff
@@ -40,11 +40,10 @@ stdenv.mkDerivation rec {
   hardeningDisable = [ "all" ];
 
   meta = {
-    broken = (stdenv.isLinux && stdenv.isAarch64);
```

Comment:
Now it is broken on intel-darwin.

Reply:
Looking at the error it seems to be an llvm compiler bug, I tried version 14 and 12 and it doesn't work with either of them, so I'm going to mark it as broken for intel darwin.